src/tasks/WasmBuildTasks/GetChromeVersions.cs

```diff
@@ -192,9 +196,45 @@ private async Task<(ChromeVersionSpec version, string baseSnapshotUrl)> FindVers
                                             foundRelease.version,
                                             foundRelease.chromium_main_branch_position.ToString(),
                                             foundRelease.version);
+        string foundV8Version = await FindV8VersionFromChromeVersion(foundRelease.version).ConfigureAwait(false);
         string? baseSnapshotUrl = await FindSnapshotUrlFromBasePositionAsync(versionSpec, throwIfNotFound: true)
                                         .ConfigureAwait(false);
+
+        versionSpec = versionSpec with { v8_version = foundV8Version };
```

Comment:
That's fine. Why don't we change the order of lines and put v8 version info into the record at once, with all the other values? But only if you will be making other serious changes, it's a nit.

Reply:
Good catch! Fixed.